Documentation/ceph-cluster-crd.md

```diff
@@ -356,6 +356,37 @@ The following storage selection settings are specific to Ceph and do not apply t
 * Mimic 13.2.3 or newer
 * Nautilus
 
+### Storage Selection Via Ceph Drive Groups
+
+Ceph supports adding devices as OSDs by Ceph Drive Group definitions in later versions of Ceph
+Octopus (v15.2.5+). Ceph Drive Groups allow for specifying highly advanced OSD layouts on nodes
+including non-homogeneous nodes. OSDs on PVCs are not supported by Drive Groups.
+See Ceph Drive Group docs for more info: https://docs.ceph.com/docs/master/cephadm/drivegroups/.
+Rook will give precedence to creating OSDs using Drive Groups before using other methods.
+
+> **IMPORTANT:** When managing a Rook/Ceph cluster's OSD layouts with Drive Groups, the `storage`
+> config is mostly ignored. `storageClassDeviceSets` can still be used to create OSDs on PVC, but
+> Rook will no longer use `storage` configs for creating OSDs on a node's devices. To avoid
+> confusion, we recommend using the `storage` config OR `driveGroups` and never both.
+
+A Drive Group is defined by a name, a Ceph Drive Group spec, and a Rook placement
+* `name`: A name for the Drive Group.
+* `spec`: The Ceph Drive group spec. Some components of the spec are treated differently in the
+  context of Rook as noted below:
+  * Ceph's definition of `placement` is ignored here in favor of Rook's `placement` below.
```

Comment:
I think this fits that description still. I need to note here that Rook overrides Ceph's definition of `placement`, `service_id`, and `host_pattern` so that users know to expect that Rook controls those spec items and that anything they specify will be overridden by Rook.

Reply:
I revised my wording to use the word `override` that I think might be a little clearer.